google_drive/tests/integration/calendar_base_test.py

```diff
@@ -0,0 +1,17 @@
+""" Google Calendar integration tests """
```

Comment:
Dumb question -- so by 'integration' test do we mean bok choy?  I see these are Selenium tests?